### DISCUSSION TITLE
GitHub Action error "Getting Resource not accessible by integration error"

### DISCUSSION THREAD

**Discussion by Pratham31** [EddieHubCommunity/support#204] [Q&A] [answered]
Actually I am developing GitHub Action but I stuck at this `ERROR` - **Resource not accessible by integration**.
Link to the my Repo - https://github.com/Pratham31/AdmireME/actions
Please help me to solve my problem I wasted two days on this error.

**Comment by eddiejaoude** [comment]
After going through your GitHub Actions [workflow](https://github.com/Pratham31/AdmireME/runs/1124370557?check_suite_focus=true), I see the following error:

<img width="806" alt="Screenshot 2020-09-16 at 19 09 29" src="https://user-images.githubusercontent.com/624760/93375605-4eb86180-f850-11ea-8b3d-4273f7d22548.png">

I think you should start at the top of the errors, with the issue is

```bash
##[warning]Unexpected input(s) 'login', valid inputs are ['query']
```

**Reply by Pratham31** [reply]
I didn't get that.

**Reply by Pratham31** [reply]
Do I need a deploy key?

**Reply by eddiejaoude** [reply]
The error is saying, you sent `login` but was expecting `query`

**Reply by Pratham31** [reply]
Will you please go through my YAML file? because It's my first ever GitHub Action. I don't know where it goes wrong.

**Reply by eddiejaoude** [reply]
I think in this [code](https://github.com/Pratham31/AdmireME/blob/master/action.yml#L41) where you have `query send($login:String!){`, it is not expecting login but `query` value

**Comment by Pratham31** [comment, marked as answer]
Resolved my Error. Actually I keep getting that error because my Email ID was private. will you please create a new issue for my repo and don't forget to make your Mail ID public.

**Reply by eddiejaoude** [reply]
glad you solved your issue. Most people will have their email private, could you reply to the Issue instead of sending them an email. They will get an email from the reply to the Issue anyway. Just an idea

**Reply by Pratham31** [reply]
Yeah, Sir. Sure 👍🏻

**Reply by Pratham31** [reply]
Thank you so much so this kind of Initiative. You are the Inspiration of too many Open Source Contributors 💯

**Reply by eddiejaoude** [reply]
Thank you for your kind words @Pratham31 👍 . Look forward to geeking out with you more in our awesome community 🤓
